src/vsr.zig

```diff
@@ -1369,12 +1372,13 @@ pub const Checkpoint = struct {
 
         const result = op: {
             if (checkpoint == 0) {
-                // First wrap: op_checkpoint_next = 8-2-1 = 5
-                break :op constants.journal_slot_count - constants.lsm_batch_multiple - 1;
+                // First wrap: op_checkpoint_next = 6-1 = 5
+                // -1: vsr_checkpoint_interval is a count, result is an inclusive index.
+                break :op constants.vsr_checkpoint_interval - 1;
             } else {
-                // Second wrap: op_checkpoint_next = 5+8-2 = 11
-                // Third wrap: op_checkpoint_next = 11+8-2 = 17
```

Comment:
I figured it would be less confusing to not decompose checkpoint interval. After all, it is morally a parameter, and can be set arbitrary, as long as it allows at least a trigger plus pipeline of slack.